packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -171,9 +177,15 @@ class Hyperlink extends React.Component {
       rel = 'noopener noreferrer';
     }
 
+    const attrSpread = {};
+    if (isDisabled) {
+      attrSpread.role = 'link';
+    }
+
     return (
-      <ComponentType
+      <a
         {...customProps}
+        {...attrSpread}
         className={hyperlinkClasses}
         aria-disabled={isDisabled}
```

Comment:
Can we move `aria-disabled` to attrSpread instead of having it on <a> element. Since it would required only for disabled scenarios. 

can we rename `attrSpread` to `attributesToDisable` OR `disabledProps` OR `additionalProps` OR `propsToDisable` OR anything that better explains what values `attrSpread` would be having.

Reply:
Done [eddebef](https://github.com/cerner/terra-core/pull/3605/commits/eddebef63386c0c1a5e7acad0c730a14e54486bb)